apps/src/templates/projects/ProjectAppTypeArea.jsx

```diff
@@ -17,26 +17,26 @@ class ProjectAppTypeArea extends React.Component {
   static propTypes = {
     labKey: PropTypes.string.isRequired,
     labName: PropTypes.string.isRequired,
-    labViewMoreString: PropTypes.string.isRequired,
+    labViewMoreString: PropTypes.string,
```

Comment:
I am not removing these props because the featured gallery may be updated in the future according to [this product spec](https://docs.google.com/document/d/1vU68tzXG72Z80MvM6xYEDwF5hLDf7V097P-0dDfpouc/edit?disco=AAABGLQtJe0).

Reply:
Could you throw a comment in this file to that effect? I feel a little torn about leaving this stuff hanging around that may or may not get used (but I'm down with the decision to keep it). I also think the "view more" button would break if it were used with the labKey of "featured" because "featured" isn't a supported `projectType` here (projectType gets copied from labKey):

https://github.com/code-dot-org/code-dot-org/blob/3342df315b85926b0401a066fed5df7867d562b2/apps/src/templates/projects/ProjectAppTypeArea.jsx#L119

So, kind of a lot of confusing stuff for future viewers to work through.

Reply:
Good points! I'll update so that a call to this api endpoint works if labKey is assigned `featured`. Will also add a comment.